src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
```

Comment:
An element is always part of a block, not another element. This means that element names can't define a hierarchy such as block__elem1__elem2.
So
Recommended: `navigation__list`
Look it here & fix in all cases
[BEM](https://en.bem.info/methodology/quick-start/#element)

```suggestion
        <ul class="navigation__list">
          <li class="navigation__item">
            <a href="#" class="navigation__link is-active">Apple</a>
          </li>
```